### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Sypheos**
Tried to write all I know about ubuild. @AlexisMontagne if you have anything to add.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  envs:
+    key: val
+    ...
+```
+
+For example this configuration will build an image based on a python package, update the release of upfluence/ner_analyser and update to the docker image named "ner-analyser".
+```yaml
+type: py
+verbose: true
+repository: <org>/<repositoryW
```
Comment (tgallice) [inline README.md]: `<repository>`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+docker:
+  image: "<image_name>"
+```
+
+## Circle CI
+To use ubuild with circle-ci you must use the golang primary image have run go get  in a step of you job. The sensitive environment variable should be set in the project settings.
```
Comment (tgallice) [inline README.md]: `get in`

File: README.md (hunk 1 above)
Comment (Sypheos) [reply README.md]:
```suggestion
repository: <org>/<repository>
```

File: README.md (hunk 2 above)
Comment (Sypheos) [reply README.md]:
```suggestion
To use ubuild with circle-ci you must use the golang primary image have run go get  in a step of you job. The sensitive environment variable should be fetched from the project settings.
```

Comment (Sypheos) [reply README.md]: @tgallice Are you ok with the suggestion ?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+# ubuild
+
+Ubuild is our docker image builder utilities. It packs several tools and workflow that we use through most of our projects. Ubuild objective is two fold, build the docker image and push it to docker hub.
```
Comment (AlexisMontagne) [inline README.md]: `ubuild` let's keep the simple case.  

and "ubuild is upfluence's release tool. It allows you to:
* Generate a release
* Compile programs
*  Build containers
* Push those containers to a remote registry

Everything in one command.

Comment (Sypheos) [reply README.md]:
```suggestion
ubuild is our release tool. It allows you to:

    Generate a release
    Compile programs
    Build containers
    Push those containers to a remote registry

Everything in one command.
```

**Review by tgallice — APPROVED** [review]

**Pull request merged**